docker/Dockerfile_dev-distccd

```diff
@@ -0,0 +1,65 @@
+FROM ubuntu:20.04
+
+LABEL maintainer="Siddharth Bharat Purohit <sidbpurohit@gmail.com>"
+LABEL author="Siddharth Bharat Purohit <sidbpurohit@gmail.com>"
+LABEL description="A distccd image based on Fedora 29 that I use for distributed compilation of ardupilot"
+
+RUN apt-get update && \
+    apt-get install -y \
+    locales
+
+RUN sed -i '/en_US.UTF-8/s/^# //g' /etc/locale.gen && \
+    locale-gen
+ENV LANG en_US.UTF-8  
+ENV LANGUAGE en_US:en  
+ENV LC_ALL en_US.UTF-8
+ENV LANG=en_US.utf8
+
+RUN apt-get install -y \
+    distcc \
+    wget
+
+ENV HOME=/home/distcc
+RUN useradd -s /bin/bash distcc
+WORKDIR /home/distcc
+RUN wget -O gcc-arm-none-eabi-9-2019-q4-major-x86_64-linux.tar.bz2 "https://developer.arm.com/-/media/Files/downloads/gnu-rm/9-2019q4/gcc-arm-none-eabi-9-2019-q4-major-x86_64-linux.tar.bz2?revision=108bd959-44bd-4619-9c19-26187abf5225&la=en&hash=E788CE92E5DFD64B2A8C246BBA91A249CB8E2D2D"
```

Comment:
just https://developer.arm.com/-/media/Files/downloads/gnu-rm/9-2019q4/gcc-arm-none-eabi-9-2019-q4-major-x86_64-linux.tar.bz2 is enought